### PR TITLE
Expose CCR to the transport client

### PR DESCRIPTION
**Pull request by jasontedor**
This commit exposes CCR to the transport client.

### CODE REVIEW

**Comment by jasontedor** [comment]
This was work that we had planned, but for a later phase. This is needed so that we can expose CCR stats via monitoring.

**Review by martijnvg — APPROVED** [review]
Assuming CI is happy - LGTM

File: x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/client/CcrClient.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            final CcrStatsAction.TasksRequest request,
+            final ActionListener<CcrStatsAction.TasksResponse> listener) {
+        client.execute(CcrStatsAction.INSTANCE, request, listener);
+    }
+
+    public ActionFuture<CcrStatsAction.TasksResponse> stats(final CcrStatsAction.TasksRequest request) {
+        final PlainActionFuture<CcrStatsAction.TasksResponse> listener = PlainActionFuture.newFuture();
+        client.execute(CcrStatsAction.INSTANCE, request, listener);
+        return listener;
+    }
```
[the remark this answers is not in the document]
Comment (jasontedor) [reply x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/client/CcrClient.java]: I pushed 512f22fd01325f246c34550ef97c6bd1091843b9.

[the remark this answers is not in the document]
Comment (martijnvg) [reply x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/client/CcrClient.java]: 👍

**Comment by jasontedor** [comment]
@elasticmachine run gradle build tests

**Pull request merged**